src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Img from './react-image'
```

Comment:
i think a succinct wat to do this is

```suggestion
export {default} from './react-image'
```